### ISSUE TITLE
Angular 1.5 .component() syntac

### ISSUE THREAD

**Issue by oilst** [kensho/ng-describe#97]
Is ng-describe supposed to work with the .component() syntax? 
I could not find any examples and could not get it running.

**Comment by oilst** [comment]
I figured out how it works. Unfortunately, it seems that the controller() function is not available for components. 
I have to call deps.element.isolateScope().$ctrl to get the controller. Is there a better way?

**Comment by bahmutov** [comment]
Yeah, we don't currently add any aliases for component syntax, need to think about it

**Comment by mikeerickson** [comment]
+1

**Comment by antgonzales** [comment]
+1

**Comment by whoadave** [comment]
Testing components is similar to testing directives. To test the component's controller, given a component named `'myComponent'`, you would access it via `var ctrl = deps.element.controller('myComponent')`

**Comment by whoadave** [comment]
Although if you just want to test the controller without testing the compiled element, it would be nice to have added support for that (can be done by injecting `$componentController` like shown here: https://puigcerber.com/2016/02/07/how-to-test-angular-1-5-components/)
